main.go

```diff
@@ -260,6 +264,15 @@ func generate(in configInput) ([]byte, error) {
 		c.Node.Metadata["TRAFFIC_DIRECTOR_CLIENT_ENVIRONMENT"] = in.deploymentInfo
 	}
 
+	if in.includeFederationSupport {
+		// Authorities with an empty server config will end up using the top-level server config.
+		// For more details, see: https://github.com/grpc/proposal/blob/master/A47-xds-federation.md#bootstrap-config-changes.
```

Comment:
Nit: Can you please ensure that comment lines are readable on narrow screens. See: go/go-style/decisions#comment-line-length.

Reply:
done 👍